apps/frontend/src/pages/ui/ui.tsx

```diff
@@ -57,6 +58,15 @@ const Ui = (): JSX.Element => {
 			<Modal title="Title" {...modal}>
 				children
 			</Modal>
+			<Breadcrumbs
+				items={[
+					{ href: "/ui", label: "Home" },
+					{ href: "/", label: "projects" },
+					{ href: "/sign-up", label: "projectname" },
+					{ label: "project details" },
```

Comment:
Even if it's just for testing, make the names consistent and logical
```suggestion
					{ href: "/", label: "GitFit" },
					{ href: "/", label: "Projects" },
					{ label: "User Interface" },
```